types/unit/timer/timerspec.pp

```diff
@@ -0,0 +1,4 @@
+# @summary Timer specification for systemd timers
+# @see https://www.freedesktop.org/software/systemd/man/systemd.timer.html
+#
+type Systemd::Unit::Timer::TimerSpec = Variant[Integer[0],String,Array[Variant[Integer[0],String]]]
```

Comment:
Should add its own rspec file in `spec/type_aliases` could just copy paste or even move the examples out of the existing timer one. Good for future  if and when its being fixed.

Reply:
done